node.js

```diff
@@ -8,30 +16,43 @@ const { Worker } = require("worker_threads");
 const PROTO_PATH = path.resolve(__dirname, "./protos/chord.proto");
 
 const packageDefinition = protoLoader.loadSync(PROTO_PATH, {
-  keepCase: true,
-  longs: String,
-  enums: String,
-  defaults: true,
-  oneofs: true
+    keepCase: true,
+    longs: String,
+    enums: String,
+    defaults: true,
+    oneofs: true
 });
 
 const chord = grpc.loadPackageDefinition(packageDefinition).chord;
 
 const caller = require("grpc-caller");
 
 const HASH_BIT_LENGTH = 3;
+const CHECK_NODE_TIMEOUT_ms = 1000;
+
+const NULL_NODE = { id: null, ip: null, port: null };
 
-const FingerTable = [
-  {
-    start: null,
-    successor: { id: null, ip: null, port: null }
-  }
+let FingerTable = [
+    {
+        start: null,
+        successor: NULL_NODE
+    }
 ];
 
-let predecessor = { id: null, ip: null, port: null };
+let SuccessorTable = [ NULL_NODE ];
 
-const _self = { id: null, ip: null, port: null };
+let _self = NULL_NODE;
 
+let predecessor = NULL_NODE;
+
+/**
+ * Accounts for the modulo arithmetic to determine whether the input value is within the bounds.
+ * Implements inclusive and exclusive properties for each bound, as specified.
+ * 
+ * @returns true if the input value is in - modulo - bounds; false otherwise
+ * 
+ * @version 20191103
```

Comment:
Appreciate these comments, but I think the version annotations are a bit unnecessary. There are tools such as Code Lens that can produce this functionality for tree without us having to do anything.